tests/e2e/testcases/localnetwork/suite.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package localnetwork
+
+import (
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Local Network]", ginkgo.Ordered, func() {
+	ginkgo.AfterEach(func() {
+		commands.CleanNetwork()
+	})
+
+	ginkgo.It("can stop a started network", func() {
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+
+		commands.StopNetwork()
+
+		out, err := commands.GetNetworkStatus()
+		gomega.Expect(err).ShouldNot(gomega.BeNil())
+		gomega.Expect(out).Should(gomega.ContainSubstring("network is not running"))
+	})
+})
```

Comment:
I know this is currently still a draft, but few thoughts: 
- we can add local network start test (and we can test the multiple flags options here)
- we can add local network status test
- use stop local network test (including the two flags that are there (snapshot, etc))

Reply:
sure can do